website/src/pages/swiss-wastewater/flu.astro

```diff
@@ -26,15 +30,21 @@ const lapisUrl = `${wastewaterConfig.lapisBaseUrl}/influenza`;
     <gs-app lapis={lapisUrl}>
         <div class='m-4 flex flex-col gap-4'>
             {
-                (['nucleotide', 'amino acid'] satisfies SequenceType[]).map((sequenceType) => (
-                    <GsWastewaterMutationsOverTime
-                        title={`${sequenceType} mutations`}
-                        lapisFilter={{
-                            versionStatus: 'LATEST_VERSION',
-                            isRevocation: false,
-                        }}
-                        sequenceType={sequenceType}
-                    />
+                references.map((reference) => (
+                    <>
+                        <h2 class='text-2xl'>{reference}</h2>
```

Comment:
Maybe we should have reused the headline component with a fragment link that let's you click on it to scroll to it?

Reply:
Ah, good point, thanks for the reminder! I can change it later.